seagull.py

```diff
@@ -9,16 +9,13 @@ class SeagullCommand(ClientPlugin):
 
 	@CommandHandler('seagull', 0)
 	def seagull(self, msg, openchar='{', closechar='}', *args):
-		if openchar == closechar:
-			self.reply(msg, "Can't make a flock out of only one character")
-			return
-		out = openchar
+		out = []
 		depth = 1
 		while depth:
-			c = openchar if random.random() * 20 > len(out) else closechar
-			if c == openchar:
+			if random.random() * 20 > len(out):
```

Comment:
This changes the target length (here `20`) to be the number of openchars/closechars, not the length of the string. This could result in huge strings with long open/close strings.